allofplos/makedb.py

```diff
@@ -155,6 +163,25 @@ class CoAuthorPLOSArticle(BaseModel):
         created_date = article.pubdate,
         word_count=article.word_count,
         JATS_type = j_type)
+    # Get subject information
+    taxonomy_set = set()
+    taxonomy = article.taxonomy
+    if taxonomy:
+        for values in taxonomy.values():
```

Comment:
If taxonomy is guaranteed to be a dict, you can remove remove the conditional.

Reply:
OK, I was thinking about an empty result, I think correction articles has no taxonomy, not sure right now, will check by running this on the whole corpus.